pkg/chunkenc/gzip_test.go

```diff
@@ -172,6 +172,37 @@ func TestGZIPSerialisation(t *testing.T) {
 	require.True(t, bytes.Equal(byt, byt2))
 }
 
+func TestGZIPChunkFilling(t *testing.T) {
+	chk := NewMemChunk(EncGZIP)
+	chk.blockSize = 1024
+
+	// We should be able to append only 10KB of logs.
```

Comment:
Might want to clarify that this is because chunks are hardcoded to allow 10 blocks per chunk.

Reply:
Or even better, should we define the number of blocks per chunk as a constant.